.travis.yml

```diff
@@ -23,6 +23,25 @@ jobs:
       script:
         - npm run lint-no-fix
         - npm run test
+      before_deploy:
+        - cd ../
+        - export TAG=travis-$TRAVIS_COMMIT-$TRAVIS_BUILD_NUMBER
+        - export ECR_REPO=$REPO-backend
+        - export PATH=$PATH:$HOME/.local/bin
+          # Install tools for deployment
+        - pip install --user awscli #For docker
+
+          # Login to AWS ECR, credentials defined in $AWS_ACCESS_KEY_ID and $AWS_SECRET_ACCESS_KEY
+        - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+        - docker build -f ./backend/Dockerfile -t $ECR_REPO:$TAG .
+        - docker tag $ECR_REPO:$TAG $ECR_REPO:$TRAVIS_BRANCH
```

Comment:
Not too sure if this additional tag is required. In our `Dockerrun.aws.json` we are running `$ECR_REPO:$TAG`, so this tagged image is never used anywhere else.

Reply:
Ah that's true. Tbh i don't really understand docker tags; I just copied Go's code. Will take another look at it.

Reply:
I think `$ECR_REPO:$TRAVIS_BRANCH` will be useful when there are multiple branches that are pushing to ECR; it makes it slightly easier to differentiate which branch pushed the image. 

Since tags are just a way for us to refer to images, seems like there is no harm in adding an extra tag.

Reply:
Yep that makes sense. I think it's useful as a quick pointer to the latest build on a particular branch. Also given that additional tags do not copy anything, there's no downside to this.